domains/primitives/beacon/src/lib.rs

```diff
@@ -174,7 +174,7 @@ pub struct InitialSync<SyncCommitteeSize: Get<u32>, ProofSize: Get<u32>> {
 )]
 #[scale_info(skip_type_params(SignatureSize, ProofSize, SyncCommitteeSize, SyncCommitteeSize))]
 #[codec(mel_bound())]
-pub struct SyncCommitteePeriodUpdate<
+pub struct LightClientUpdate<
```

Comment:
Why the change here ?

Reply:
https://github.com/subspace/subspace/pull/1170#issuecomment-1453590351